resources/sushy-tools/Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM registry.hub.docker.com/library/python:3.9
 
+ENV SUSHY_TOOLS_VERSION="0.18.1"
+
 RUN apt update && \
     apt install -y libvirt-dev && \
-    pip3 install sushy-tools==0.18.1 libvirt-python
+    apt clean && \
+    pip3 install --no-cache-dir \
+        sushy-tools==${SUSHY_TOOLS_VERSION} libvirt-python
+
+ENV SUSHY_EMULATOR_EXTRA_FLAGS="--debug"
```

Comment:
let's move this under the first ENV and compress to a single ENV

Reply:
If finally using `SUSHY_TOOLS_VERSION` as ARG (see the other comment), I guess it wouldn't make sense to have them both as ENV.

Reply:
of course not, this comment is about a change that should be applied now, considering that both are ENVs
the other comment was about a possible follow-up, to not add complexity to this change
all considered "--debug" does not need to be an ENV in this change

Reply:
I think I got it. I've created an ARG for the `sushy-tools` version and an ENV for the extra flags. Do you mind to take a look again? Thanks!

Reply:
the SUSHY_TOOLS_VERSION ARG is ok, but what I was really saying is that I'm not sure SUSHY_EMULATOR_EXTRA_FLAGS should be an ENV, you're creating one more layer for a thing that could be included in the CMD and does not need to persist in other parts of the Dockerfile

Reply:
ack